app/controllers/external_authors_controller.rb

```diff
@@ -74,7 +73,10 @@ def update
       @external_author.delete_works
       flash[:notice] += "Your imported stories have been deleted. "
     end
-    @invitation.mark_as_redeemed if @invitation && !params[:imported_stories].blank?
+
+    if @invitation && params[:imported_stories].present? && params[:imported_stories] != "nothing"
```

Comment:
Justification of this style change: it was too gosh darn long to keep in the one-line format.

Reply:
FYI you can break the conditional into multiple lines as well, like:

https://github.com/otwcode/otwarchive/blob/bedbbef8c43585d97b5a56062355921e4c8f5fa7/app/models/creatorship.rb#L147-L149